Source/WebCore/platform/graphics/cv/ImageRotationSessionVT.mm

```diff
@@ -138,17 +138,17 @@
 
 RetainPtr<CVPixelBufferRef> ImageRotationSessionVT::rotate(VideoFrame& videoFrame, const RotationProperties& rotation, IsCGImageCompatible cgImageCompatible)
 {
-    auto pixelBuffer = videoFrame.pixelBuffer();
+    RetainPtr pixelBuffer = videoFrame.pixelBuffer();
     ASSERT(pixelBuffer);
     if (!pixelBuffer)
         return nullptr;
 
-    m_pixelFormat = CVPixelBufferGetPixelFormatType(pixelBuffer);
-    IntSize size { (int)CVPixelBufferGetWidth(pixelBuffer), (int)CVPixelBufferGetHeight(pixelBuffer) };
+    m_pixelFormat = CVPixelBufferGetPixelFormatType(pixelBuffer.get());
+    IntSize size { (int)CVPixelBufferGetWidth(pixelBuffer.get()), (int)CVPixelBufferGetHeight(pixelBuffer.get()) };
```

Comment:
I know this is pre-existing code but can we use `static_cast<int>` for `(int)`?

Reply:
Ok.